client/client.ts

```diff
@@ -25,7 +41,7 @@ async function editlisting (): Promise<void> {
 	const category = $('#categories').find(':selected').val();
 	const subcategory = $('#subcategories').find(':selected').val();
 	const currentlistingquery = location.search;
-	const currentlisting = currentlistingquery.substring(11);
+	const currentlisting = currentlistingquery.substring(4);
```

Comment:
magic number previously removed ?listingId= which was 11 chars and now it removes ?id= which is 4 chars because location.search takes everything after /item or /edit which contains the listing id in base64 format in the query